src/components/@common/embeded-frame.tsx

```diff
@@ -0,0 +1,59 @@
+import { useState, useRef, useEffect } from "react";
+
+import Button from "./button";
+import LoadingSpinner from "./loading-spinner";
+
+export default function EmbeddedFrame({ linkUrl }: { linkUrl: string }) {
+  const iframeRef = useRef<HTMLIFrameElement>(null);
+  const [isBlocked, setIsBlocked] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const checkLinkStatus = async (url: string) => {
+    try {
+      const response = await fetch(url, {
+        method: "HEAD",
+        redirect: "follow",
+      });
+      if (!response.ok) {
+        throw new Error(`HTTP error! status: ${response.status}`);
+      }
+      setIsBlocked(false);
+    } catch (error) {
+      setIsBlocked(true);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    checkLinkStatus(linkUrl);
+  }, [linkUrl]);
+
+  return (
+    <div className="right-800 top-0 z-50 mt-24 flex h-372 w-full items-center justify-center bg-slate-200 text-center lg:absolute lg:m-0 lg:h-full lg:w-543">
+      <div className="flex h-full w-full items-center justify-center lg:h-[413.56px]">
+        {isLoading ? (
+          <LoadingSpinner />
+        ) : isBlocked ? (
+          // 임베디드 차단
+          <div className="flex h-full w-full flex-col items-center justify-center bg-white bg-opacity-30 lg:bg-opacity-100">
+            <p className="mb-20 text-slate-500">
+              이 사이트는 임베디드할 수 없습니다.
+            </p>
```

Comment:
'embeded'는 수동형이라서 '임베디드할 수 없다'는 표현이 조금 어색하게 느껴지는 것 같아요. '임베드할 수 없다'고 표현하는 것이 조금 더 적절한 표현 같은데, 어떻게 생각하시나요?
```suggestion
            <p className="mb-20 text-slate-500">
              이 링크는 임베드할 수 없습니다.
            </p>
```